src/lib/utils.ts

```diff
@@ -61,3 +47,48 @@ export function getType(obj: any) {
 
   return false;
 }
+
+/**
+ * Determine if a given schema if an OpenAPI definition.
+ *
+ */
+export function isOpenAPI(schema: Record<string, unknown>) {
+  return !!schema.openapi;
+}
+
+/**
+ * Determine if a given schema is a Postman collection.
+ *
+ * Unfortunately the Postman schema spec doesn't have anything like `openapi` or `swagger` for us
+ * to look at but it does require that `info` and `item` be present and as `item` doesn't exist in
+ * OpenAPI or Swagger we can use the combination of those two properties to determine if what we
+ * have is a Postman collection.
+ *
+ * @see {@link https://schema.postman.com/json/collection/v2.0.0/collection.json}
+ * @see {@link https://schema.postman.com/json/collection/v2.1.0/collection.json}
+ */
+export function isPostman(schema: Record<string, unknown>): boolean {
+  return !!schema.info && !!schema.item;
```

Comment:
Kind of frustrating that the Postman spec doesn't have a `version` property that says if it's for v2.0 or 2.1. Like, come on.

Reply:
wow i hate this so much 😮‍💨 what about checking for the existence of `info._postman_id`? I feel like that's always a thing?

Reply:
`info._postman_id` isn't actually required, just `info.name` and `info.schema`. 😠 

![Screen Shot 2022-11-07 at 4 00 47 PM](https://user-images.githubusercontent.com/33762/200441191-f0a56bfa-dfa7-4569-a525-a68f58e76fc0.png)

I'm going to begrudgingly keep this as-is.